web/api/report_server.thrift

```diff
@@ -724,4 +729,14 @@ service codeCheckerDBAccess {
   AnalyzerStatisticsData getAnalysisStatistics(1: i64 runId,
                                                2: i64 runHistoryId)
                                                throws (1: codechecker_api_shared.RequestFailed requestError),
+  
+  // Export data from the server
+  // PERMISSION: PRODUCT_ACCESS
+  ExportData exportData(1: i64 runId)
```

Comment:
Unfortunately this API function does not allow to filter by multiple run names which I think should be supported. For this reason we need to change this API function to this one:
```suggestion
  ExportData exportData(1: RunFilter runFilter)
```
For this I created the new packages: `6.37.0-dev5`.